mix.exs

```diff
@@ -30,13 +39,18 @@ defmodule Bigben.Mixfile do
   #
   # Type `mix help deps` for examples and options.
   defp deps do
-    [{:phoenix, "~> 1.1.4"},
-     {:postgrex, ">= 0.0.0"},
-     {:phoenix_ecto, "~> 2.0"},
-     {:phoenix_html, "~> 2.4"},
-     {:phoenix_live_reload, "~> 1.0", only: :dev},
-     {:gettext, "~> 0.9"},
-     {:cowboy, "~> 1.0"}]
+    [
+      {:cowboy, "~> 1.0"},
+      {:ex_machina, "~> 0.6.1"},
+      {:gettext, "~> 0.9"},
+      {:phoenix, "~> 1.1.4"},
+      {:phoenix_ecto, "~> 2.0"},
+      {:phoenix_html, "~> 2.4"},
+      {:phoenix_live_reload, "~> 1.0", only: :dev},
+      {:postgrex, ">= 0.0.0"},
+      {:timex, "~> 1.0.1"},
+      {:timex_ecto, "~> 0.9.0"},
+   ]
```

Comment:
I actually started to put in a PR today to phoenix on that. I think I'm going to do it now that you've prompted.